cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/service/http/HttpHandlerGenerator.java

```diff
@@ -542,6 +546,14 @@ private void generateTransactionalDelegateBody(GeneratorAdapter mg, Method metho
                        Methods.getMethod(TransactionContext.class, "getTransactionContext"));
       mg.storeLocal(txContext, txContextType);
 
+      // HttpServiceResponder wrappedResponder = wrapResponder(responder);
+      int wrappedResponder = mg.newLocal(serviceResponderType);
+      mg.loadThis();
```

Comment:
I am not familiar with this code... what is mg? Can we rename it something more meaningful?